scripts/release/generate_history_notes.py

```diff
@@ -115,7 +116,13 @@ def modify_history_file(file: fileinput.FileInput, new_history: str):
 
 def construct_cli_history(component: str):
     history = '**{}**\n\n'.format(component)
+    non_breaking_change_notes = []
     for note in history_notes[component]:
+        if 'BREAKING CHANGE' in note.upper():
```

Comment:
how to force the string format from beginning?

Reply:
You mean in the PR? Currently, PR reviewers are also responsible for reviewing the history note format and a double-check is performed during release pipeline.